src/library/F3DOptions.cxx

```diff
@@ -197,7 +181,7 @@ F3DOptions ConfigurationOptions::GetOptionsFromArgs(std::vector<std::string>& in
     this->DeclareOption(grp1, "edges", "e", "Show cell edges", options.Edges);
     this->DeclareOption(grp1, "trackball", "k", "Enable trackball interaction", options.Trackball);
     this->DeclareOption(grp1, "progress", "", "Show progress bar", options.Progress);
-    this->DeclareOption(grp1, "up", "", "Up direction", options.Up, true, "[-X|+X|-Y|+Y|-Z|+Z]");
+    this->DeclareOption(grp1, "up", "", "Up direction", options.Up, true, true, "[-X|+X|-Y|+Y|-Z|+Z]");
```

Comment:
how could this even work before ?

Reply:
It wasn't working

Reply:
`arg` was displayed instead of `[-X|+X|-Y|+Y|-Z|+Z]`